### DISCUSSION TITLE
S4ge_Rites Questions

### DISCUSSION THREAD

**Discussion by ghost** [ASEAN-Build-The-Earth/Merlion#41] [Q&A] [answered]
Im currently doing the contribution

@S4ge-Rites  - admin edit please specify more and your question title dosnt tell us anything so please specify more.

**Comment by StoneMcYT** [comment]
No idea what you are asking.

**Comment by ghost** [comment]
its not a question its so that i can start the convo

**Comment by StoneMcYT** [comment, marked as answer]
Locked as it'd not a question m
